CHANGELOG.md

```diff
@@ -1,8 +1,15 @@
 ﻿# Cities: Skylines - Traffic Manager: *President Edition* [![Discord](https://img.shields.io/discord/545065285862948894.svg)](https://discord.gg/faKUnST)
 
 # Changelog
-1.10.16, [Work in progress]
-
+1.10.16, 24/02/2019
+- Gameplay: Fixed problem with vehicles unable to choose lane when u-turn at dead-end (thanks @pcfantasy for implementation and @aubergine10 for neccesary tests)(#101)
+- Gameplay: Fixed problem with vehicle despawn after road upgrade/remove (thanks @pcfantasy for implementation suggestion)(#86, #101)
```

Comment:
I'd move this one up to top - the despawn problem was a bigger annoyance for users.